.github/workflows/image_release.yaml

```diff
@@ -0,0 +1,127 @@
+name: Gingersnap Operator - Build and publish
+
+on:
+  push:
+    branches:
+      - 'Branch Name'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}   
+  release-version: "0.0.0"
```

Comment:
This needs to be set by an input. To minimise the changes to the rest of the action, we could doo:

`release-version: ${{ github.event.inputs.release }}`